src/stores/Video.js

```diff
@@ -193,19 +193,35 @@ class VideoStore {
       if(!this.isVideo) {
         this.initialized = true;
       } else {
-        this.availableOfferings = yield this.rootStore.client.AvailableOfferings({
+        const offerings = yield this.rootStore.client.AvailableOfferings({
           versionHash: videoObject.versionHash
-        });
+        }) || {};
+        const offeringPlayoutOptions = {};
+        const browserSupportedDrms = (yield this.rootStore.client.AvailableDRMs() || []).filter(drm => ["clear", "aes-128"].includes(drm));
+
+        for(let offering of Object.keys(offerings).sort()) {
```

Comment:
this should explicitly attempt to use the `default` offering, if available. only pick from other options if default is not possible